package.json

```diff
@@ -15,9 +15,10 @@
     "packages/indexer"
   ],
   "scripts": {
-    "bootstrap": "yarn workspaces foreach -Apv --exclude @nomad-xyz/local-environment --exclude @nomad-xyz/monitor run bootstrap",
-    "build": "yarn workspaces foreach -Atv  --exclude @nomad-xyz/local-environment --exclude @nomad-xyz/monitor run build",
-    "compile": "yarn workspaces foreach -Apv --exclude @nomad-xyz/local-environment --exclude @nomad-xyz/monitor run compile",
+    "compile": "yarn workspaces foreach -Apv --exclude @nomad-xyz/deploy --exclude @nomad-xyz/local-environment --exclude @nomad-xyz/monitor run compile",
+    "bootstrap": "yarn build:accumulator-cli && yarn workspaces foreach -Atv --exclude @nomad-xyz/deploy --exclude @nomad-xyz/local-environment --exclude @nomad-xyz/monitor run bootstrap",
+    "build": "yarn build:accumulator-cli && yarn workspaces foreach -Atv --exclude @nomad-xyz/deploy --exclude @nomad-xyz/local-environment --exclude @nomad-xyz/monitor run build",
+    "build:accumulator-cli": "curl https://sh.rustup.rs -sSf | sh -s -- -y -c cargo && cd scripts && ./build-acli.sh",
```

Comment:
move the prepwork into the build-acli script. avoid re-curling if they already have the toolchain